### ISSUE TITLE
Rule 191 reports false negative for package path on Windows

### ISSUE THREAD

**Issue by chkohner** [googleapis/api-linter#864]
Looks like the offending code is not properly using the platform's path separator.

A potential fix, in `rules/aip0191/proto_package.go`:

```go
dir := filepath.Dir(f.GetName())
pkg := strings.ReplaceAll(f.GetPackage(), ".", string(filepath.Separator)) // <-- was incorrectly using "/"
```
